syntax = "proto3";

package Micro.Future.Message.Business;

message BizErrorMsg {
	uint32 serialId = 1;
	uint32 messageId = 2;
	int32 errorcode = 3;
	bytes description = 4;
	int32 syserrcode = 5;
}

message Result
{
	int32 code = 1;
	uint32 serialId = 2;
}

message RawData
{
	bytes data = 1;
}